src/sentry/integrations/vsts/notify_action.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import
+
+import logging
+import sentry_sdk
+
+from django import forms
+
+from sentry.models import Integration
+from sentry.rules.actions.base import EventAction
+from sentry.utils import metrics
+
+logger = logging.getLogger("sentry.rules")
+
+
+class AzureDevopsNotifyServiceForm(forms.Form):
+    # TODO 2.0 Add form fields.
+
+    def __init__(self, *args, **kwargs):
+        super(AzureDevopsNotifyServiceForm, self).__init__(*args, **kwargs)
+
+    def clean(self):
+        return super(AzureDevopsNotifyServiceForm, self).clean()
+
+
+class AzureDevopsCreateTicketAction(EventAction):
+    form_cls = AzureDevopsNotifyServiceForm
+    label = u"TODO Create a {name} AzureDevops workitem"
+    prompt = "Create a AzureDevops workitem"
+
+    def __init__(self, *args, **kwargs):
+        super(AzureDevopsCreateTicketAction, self).__init__(*args, **kwargs)
+        # TODO 2.1 Add form_fields
+        self.form_fields = {}
+
+    def is_enabled(self):
+        # TODO move to superclass
+        return self.get_integrations().exists()
+
+    def after(self, event, state):
+        integration_id = None
```

Comment:
@mgaeta should this be part of the PR if this isn't actually going to be implemented?

Reply:
I'll pull it out.